README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
Clicking "Reset" in the Scratch editor doesn't delete variables from the UI variable list. The variables do seem to be deleted based on the XML, and the UI eventually updates when other variables are changed